extensions/yafw/package.json

```diff
@@ -22,6 +25,88 @@
       "title": "Open History",
       "description": "See videos you've recently compressed",
       "mode": "view"
+    },
+    {
+      "name": "convert",
+      "title": "Convert To",
+      "description": "The command takes no arguments and encodes the file in the selected folder.",
+      "mode": "no-view",
+      "arguments": [
+        {
+          "name": "format",
+          "placeholder": "Format",
+          "type": "dropdown",
+          "data": [
+            {
+              "title": "MP4",
+              "value": "mp4"
+            },
+            {
+              "title": "WEBM",
+              "value": "webm"
+            },
+            {
+              "title": "GIF",
+              "value": "gif"
+            }
+          ],
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "optimize",
+      "title": "Optimize",
+      "description": "The command allows you to optimize the selected video according to the selected template. There are 3 templates available: lighter weight, optimal and best quality. Under the hood, these settings affect the bitrate. GIFs are not currently supported for optimization.",
```

Comment:
Yes, because ffmpeg uses different algorithms to create gif files. When converting video it works well, but when converting gif files it returns huge files. I tested on a 10 mb mp4 file and got a 2 mb gif. Then I transferred that gif to optimize and it returned me a 70mb file 😅.
But I think we can explore further if this is feasible. If so, I think it would be better to create a separate PR for this, what do you think?

Reply:
That's the behavior I get in other apps (that probably use ffmpeg). Let's leave it for  future research

Reply:
I left the comment regarding this issue in the code